### ISSUE TITLE
Integrate Recommended Actions with Github Issues

### ISSUE THREAD

**Issue by vpetersson** [WoTTsecurity/api#505]
tl;dr: Push recommended actions to Github.

Part of the value proposition we're trying to fix the workflow of solving security issues. We're testing this with a Github Issues integration. 

Here's a flowchart to illustrate the concept:

![Github Issue Integration(2)](https://user-images.githubusercontent.com/357664/68395614-f7ddf800-0167-11ea-94d2-d0b0339711db.png)

(Consider this ticket a meta-ticket, as it needs to be broken down into smaller parts.)

## Authentication with Github

Before we can access a 3rd party's Github repository, we need the user to grant us access. This is likely best done using the [Github Apps](https://developer.github.com/apps/) feature. At the end of the process, we should receive an oAuth token that we can then use to interact with the Issues API.

As part of this, we also need the user to select what repository they want the issues created in.

## Interacting with Issues

The main end-point that we need to interact with is the [Issues](https://developer.github.com/v3/issues/) and [Comments](https://developer.github.com/v3/issues/comments/) API. We need to do three things:

 * Create issue
 * Close Issue
 * Post comments to an issue

## Workflow explained

(We're skipping the auth section).

Let's say we have generated the following recommended action:

```
Insecure services found

We found insecure services installed on wott1. Because these services are considered insecure, it is recommended that you uninstall them.

Run sudo apt-get purge rsh-server to disable all insecure services.
```

The first thing we need to do is to create a Github issue for this in selected repository. Note that we need to use proper markdown for this and it should be signed-off using the WoTT Bot (the user we should use for the app above).

Once we have create the issue, we need a celery task that runs every 6 hours to check on the issue. If it has been resolved, we will automatically close the issue.

Note that we also need to link to the relevant Gihub issue in the Recommended Action in the WoTT dashboard.
